### ISSUE TITLE
Cryptic error "Containerfile: no such file or directory"

### ISSUE THREAD

**Issue by rommeswi** [Vanilla-OS/custom-image#26]
After editing the recipe.yml, the github actions started showing the error:
"Error: buildx failed with: ERROR: failed to solve: failed to read dockerfile: open Containerfile: no such file or directory"

This error message did not give any indication where the error was.
The code passed the 'verify_image' and 'check_update' stages.

I am still not sure what caused the error to occur. I mainly added a file to /modules/ that contains a shell module to download a bunch of fonts. Including this script in the recipe caused the problem.

My guess is that generating the containerfile failed and lead to the error. Ideally, the build script gives an indication where the build of the containerfile failed.

**Comment by kbdharun** [comment]
Hi, If possible, can you share a link to your repository for further debugging?

**Comment by mirkobrombin** [comment]
That happen when vib fails to generate the Containerfile which means the recipe could be broken. As Dharun asked, provide the repo url please.

**Comment by rommeswi** [comment]
I fixed the error by outsourcing the commands to a bash script. I think it was just a syntax problem in the .yml of the form:

```
name: bla
type: shell
commands:
- "echo unescaped " messed up my script"
```

The main issue is that a missing dockerfile error is not very indicative of the build script problem. (It could have been that the specified image to build on was mistyped)

**Comment by mirkobrombin** [comment]
@kbdharun is it possible to make the step after vib, to check for that fail and provide a better failure log?

**Comment by kbdharun** [comment]
> @kbdharun is it possible to make the step after vib, to check for that fail and provide a better failure log?

Yeah, we can make it show up in the workflow's annotations will look into it.
